src/libp2p/connectionsManager.ts

```diff
@@ -138,39 +147,41 @@ export class ConnectionsManager {
   }
 
   public initializeNode = async (staticPeerId?: PeerId): Promise<ILibp2pStatus> => {
+    console.log('local address:', this.localAddress)
+    console.log('bootstrapMultiaddrs:', this.bootstrapMultiaddrs)
+
     if (!staticPeerId) {
       this.peerId = await this.getPeerId()
     } else {
       this.peerId = staticPeerId
     }
     this.createAgent()
+    this.localAddress = `${this.listenAddrs}/p2p/${this.peerId.toB58String()}`
```

Comment:
linter won't allow using array in string expression, but it's already fixed no 'add-test' branch

Reply:
fixed